app/build.gradle

```diff
@@ -77,6 +77,37 @@ def cordaCLi = tasks.named('jar', Jar) {
     archiveVersion = project.version
 }
 
+// Proposed alternative to publishOSGiImage
+// Currently in trial phase, being rolled out in Nightly and locally and planned to be deployed in phases down the line
+tasks.register('publishBuildkitImage', BuildkitBuild) {
+    sourceTasks = Arrays.asList(cordaCLi.get())
+    dependsOn(cordaCLi)
+    overrideEntryName = 'cli'
+    containerName = 'cli'
+    arguments = Arrays.asList("-Dpf4j.pluginsDir=/opt/override/plugins")
+    if (project.hasProperty('baseImage')) {
+        baseImageName = baseImage
+    }
+    if (project.hasProperty('isNightly')) {
+        nightlyBuild = isNightly.toBoolean()
+    }
+    if (project.hasProperty('workerBaseImageTag')) {
+        baseImageTag = workerBaseImageTag
+    }
+    if (project.hasProperty('useBuildx')) {
+        it.isBuildx = useBuildx.toBoolean()
+    }
+    if (project.hasProperty('tag')) {
+        it.containerTag = tag
+    }
+    if (project.hasProperty('repo')) {
+        it.containerRepo = repo
+    }
+    if (project.hasProperty('isDocker')) {
```

Comment:
change `isDocker` to `publishToDockerDaemon` for clarity. - of course make this same change anywhere this is called (shared lib / runtime-os)